src/test/java/org/programmers/springbootbasic/voucher/VoucherServiceTest.java

```diff
@@ -0,0 +1,92 @@
+package org.programmers.springbootbasic.voucher;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.programmers.springbootbasic.voucher.model.FixedAmountVoucher;
+import org.programmers.springbootbasic.voucher.model.PercentDiscountVoucher;
+import org.programmers.springbootbasic.voucher.model.Voucher;
+import org.programmers.springbootbasic.voucher.model.VoucherType;
+import org.programmers.springbootbasic.voucher.repository.VoucherRepository;
+import org.programmers.springbootbasic.voucher.service.VoucherService;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.*;
+
+class VoucherServiceTest {
+
+    @Mock
+    VoucherRepository voucherRepository = mock(VoucherRepository.class);
```

Comment:
```suggestion
    @Mock
    private VoucherRepository voucherRepository;
```

모킹 하려면 이렇게 해도 되겠는걸요?
만약에 초기화 까지 하신다면 @Mock이 필요 없을 것 같습니다.